server/data/incidentClient.ts

```diff
@@ -280,4 +282,36 @@ export default class IncidentClient {
       values: [formResponse, incidentDate, reportId],
     })
   }
+
```

Comment:
there's a draftReportClient.ts which is a place for methods involving reports before they are submitted

Reply:
moved function from incidentClient to draftReportClient

Reply:
done